
field_width: 42.40
field_depth: 35.20

line_separation: 0.52
line_num: 76

headland: 9.0